kubetest/e2e.go

```diff
@@ -604,6 +604,10 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		return control.FinishRunning(exec.Command("./test/kubemark/stop-kubemark.sh"))
 	})
 
+	if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
```

Comment:
Not sure, I understand the motivation of doing this?
Don't we check already before entering this function for IsUp?

Reply:
unfortunately nope - for regular e2e test, IsUp is called from https://github.com/kubernetes/test-infra/blob/master/kubetest/e2e.go#L225 (before the ginkgo call), but not for kubemarktest

Reply:
Ok, that sgtm.